packages/xo-web/src/common/intl/messages.js

```diff
@@ -791,6 +791,7 @@ const messages = {
     'RAM: {memoryUsed} used on {memoryTotal} ({memoryFree} free)',
   hardwareHostSettingsLabel: 'Hardware',
   hyperThreading: 'Hyper-threading (SMT)',
+  hyperThreadingNotAvailable: 'HT detection is only available on XCP-ng',
```

Comment:
in XCP-ng?

Reply:
It depends on where the information can be found. If you are directing someone to look in a book or brochure, then "the information is available in the brochure." 

If one must go somewhere to get the information, then "the information is available at the main office."

Finally, if the information is attached to something else, such as a registration number, then "the information is available on the bar code tag located on the bottom of the computer."